### PR TITLE
 - Created HW3

### PR DESCRIPTION
**Pull request by tzatushevskaya**
- Added images
- Added poisonous plants
- Added a table
- Updated README

### CODE REVIEW

**Comment by eclimov** [comment]
@tzatushevskaya , please, add reviewers so we could review this pull request

**Comment by tzatushevskaya** [comment]
@eclimov done

File: public/Sjofn/homework3.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<body>
+<table style="width:100%">
+    <tr>
+        <th>Nr</th>
+        <th>Name</th>
+        <th>Picture</th>
+        <th style="width: 30%">Description</th>
+    </tr>
+    <tr>
+        <td>I</td>
```
Comment (eclimov) [inline public/Sjofn/homework3.html]: Please, use arabic numerals.

File: public/Sjofn/homework3.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    <tr>
+        <td>VI</td>
+        <td>Red magic lily</td>
+        <td><img src="./assets/redmagiclily.jpg"></td>
+        <td>Lycoris radiata, known as red spider lily, hell flower, red magic lily, or equinox flower, is a plant in the amaryllis family, Amaryllidaceae, subfamily Amaryllidoideae. Originally from China, Korea, and Nepal, it was introduced into Japan and from there to the United States and elsewhere. It is considered naturalized in Seychelles and in the Ryukyu Islands.[3] It flowers in the late summer or autumn, often in response to heavy rainfall. The common name hurricane lily refers to this characteristic, as do other common names, such as resurrection lily; these may be used for the genus as a whole.</td>
+    </tr>
+    <tr>
+        <td>VII</td>
+        <td>Wisteria</td>
+        <td><img src="./assets/wisteria.jpg"></td>
```
Comment (eclimov) [inline public/Sjofn/homework3.html:102]: You forgot to add the image to VCS (Version Control System)

Comment (eclimov) [reply public/Sjofn/homework3.html:102]: wisteria_missing_image

File: public/Sjofn/homework3.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            border-collapse: collapse;
+        }
+        tr:nth-child(even) {
+            background-color: #f2f2f2;
+        }
+        td {
+            text-align: center;
+        }
+        tr:not(:first-child):hover {
+            background: url('./assets/bg.jpg') repeat;
```
Comment (eclimov) [inline public/Sjofn/homework3.html]: This image doesn't quite fit here, so I suggest you to use a more pale variant of it.
I added and committed an example for you into your branch: 'bg_pale.png', - try using it instead.

File: public/Sjofn/homework3.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+            opacity: 100%;
```
Comment (eclimov) [inline public/Sjofn/homework3.html]: You don't need this property. It can be removed.

File: README.md [hunk 5]
```diff
@@ -54,3 +54,17 @@ Add 2 input fields with placeholders ‘a’ and ‘b’ accordingly.
...
+1. Create branch ‘<username>-hw3’. Make the following changes:
+    1. Update 'README.md' with task of this homework
+    2. Implement a HTML page with table of TOP 5 favorite flowers. Columns: 
+        - № (just an index number from 1 to 5)
+        - Name
+        - Picture (store pictures locally, - create a folder called ‘assets’)
+        - Description (a short description of the flower)
+    3. Make a ‘zebra’ view of the table
+    4. Add a CSS style that changes background color of a table row’s on hover
+    5. Use palette when picking colors
```
Comment (eclimov) [inline README.md:68]: Please, implement this point as well. 
You may pick any palette and set some custom colors to, for example:
- background color (of the table or the whole 'body')
- table's border 
- text
- table's zebra

**Review by SandaPanteleimonov — APPROVED** [review]
Awesome work, background image is stunning!

**Review by eclimov — APPROVED** [review]
Approved. Good job!

**Pull request merged**